plugins/inputs/zfs/zfs_freebsd.go

```diff
@@ -8,6 +8,7 @@ import (
 	"os/exec"
 	"strconv"
 	"strings"
+	"log"
```

Comment:
You need to remove this otherwise golang will generate "unused import" warnings/errors.

Reply:
You can mark this as resolved.